huaweicloud/services/ddm/resource_huaweicloud_ddm_account.go

```diff
@@ -0,0 +1,518 @@
+					validation.StringLenBetween(1, 32),
+				),
+			},
+			"password": {
+				Type:        schema.TypeString,
+				Required:    true,
+				Sensitive:   true,
+				Description: `Specifies the DDM account password.`,
+			},
+			"permissions": {
+				Type: schema.TypeList,
+				Elem: &schema.Schema{
+					Type: schema.TypeString,
+					ValidateFunc: validation.StringInSlice([]string{
+						"CREATE", "DROP", "ALTER", "INDEX", "INSERT", "DELETE", "UPDATE", "SELECT",
+					}, false),
+				},
+				Required:    true,
+				Description: `Specifies the basic permissions of the DDM account.`,
+			},
+			"description": {
+				Type:        schema.TypeString,
+				Optional:    true,
+				Computed:    true,
+				Description: `Specifies the description of the DDM account.`,
+			},
+			"schemas": {
+				Type:        schema.TypeList,
+				Elem:        AccountSchemaSchema(),
+				Optional:    true,
+				Computed:    true,
+				Description: `Specifies the schemas that associated with the account.`,
+			},
+			"status": {
+				Type:        schema.TypeString,
+				Computed:    true,
+				Description: `Indicates the status of the DDM account.`,
+			},
+		},
+	}
+}
+
+func AccountSchemaSchema() *schema.Resource {
+	sc := schema.Resource{
+		Schema: map[string]*schema.Schema{
+			"name": {
+				Type:        schema.TypeString,
+				Optional:    true,
+				Computed:    true,
+				Description: `Specifies the name of the associated schema.`,
+			},
+			"description": {
+				Type:        schema.TypeString,
+				Computed:    true,
+				Description: `Specifies the schema description.`,
+			},
+		},
+	}
+	return &sc
+}
+
+func resourceDdmAccountCreate(ctx context.Context, d *schema.ResourceData, meta interface{}) diag.Diagnostics {
+	cfg := meta.(*config.Config)
+	region := cfg.GetRegion(d)
+
+	// createAccount: create DDM account
+	var (
+		createAccountHttpUrl = "v1/{project_id}/instances/{instance_id}/users"
+		createAccountProduct = "ddm"
+	)
+	createAccountClient, err := cfg.NewServiceClient(createAccountProduct, region)
+	if err != nil {
+		return diag.Errorf("error creating DDM Client: %s", err)
+	}
+
+	instanceID := d.Get("instance_id").(string)
+	createAccountPath := createAccountClient.Endpoint + createAccountHttpUrl
+	createAccountPath = strings.ReplaceAll(createAccountPath, "{project_id}", createAccountClient.ProjectID)
+	createAccountPath = strings.ReplaceAll(createAccountPath, "{instance_id}", fmt.Sprintf("%v", instanceID))
+
+	createAccountOpt := golangsdk.RequestOpts{
+		KeepResponseBody: true,
+		OkCodes: []int{
+			200,
+		},
+	}
+	createAccountOpt.JSONBody = utils.RemoveNil(buildCreateAccountBodyParams(d))
+
+	var createAccountResp *http.Response
+	for {
+		createAccountResp, err = createAccountClient.Request("POST", createAccountPath, &createAccountOpt)
+		if err == nil {
+			break
+		}
+		err = handleOperationError(ctx, d, cfg, err, instanceID, schema.TimeoutCreate)
+		if err != nil {
+			return diag.Errorf("error creating DdmAccount: %s", err)
```

Comment:
```suggestion
			return diag.Errorf("error creating DDM account: %s", err)
```
please update them in all files

Reply:
done